notebooks/README.md

```diff
@@ -0,0 +1,10 @@
+# Growth Model Notebooks
+
+The file contains the notebooks used for developing growth model. The following list describing each of them. 
+
+- [growth_model_eda.ipynb](../growth_model_eda.ipynb) preprocesses the public dataset from [Kaggle](https://www.kaggle.com/datasets/census/business-and-industry-reports?select=notes.txt) to be used for training an Auto ARIMA time series forecasting model.
+- [growth_model_forecast.ipynb](../growth_model_forecast.ipynb) develops a time series forecasting model on the pre-processed dataset to forecast future revenues.
+- [growth_model_classification.ipynb](../growth_model_classification.ipynb) classifies customers based on their potential to grow by using the forecasted data. 
+- [helper_functions.ipynb](../helper_functions.ipynb) provides a set of helper functions that can be used to perform various tasks in the analysis.
```

Comment:
Include a `-` after the notebook name

Reply:
maybe reword it to: `.....that are used in other notebooks to perform various tasks`

Reply:
We can also remove the `.` at the end of each description